src/presentation/facades/controller-facade.ts

```diff
@@ -6,14 +6,16 @@ import { BaseController } from '../controllers/base-controller'
 import { TimelineController } from '../controllers/timeline-controller'
 import { ActionController } from '../controllers/action-controller'
 import { ConfigurationController } from '../controllers/configuration-controller'
+import {IngestController} from '../controllers/ingest-controller'
```

Comment:
Has Lint been applied?

Reply:
I don't think the rules you are thinking about are enforced on sofie-server yet :-)